cwd/views/editor.pug

```diff
@@ -0,0 +1,32 @@
+doctype html
+html
+  head
+    title CWD Editor
+    link(href="./public/styleSheet.css" rel="stylesheet" type="text/css")
+
+    body
+      <div id="pass">
+      <p> Password: <input type="password" id="passIn"></input></p>
+      </div>
+
+      <div id="glscButton">
```

Comment:
These buttons need to be dynamically generated.  Not all dashboards will have these views.

Reply:
I will bring up this issue again if it is still an issue once we are using React.  For now, I think it's no big deal.